BUILDING.rst

```diff
@@ -123,8 +123,12 @@ Windows
   that you have the older 1.108 release and **not** the newer 1.114
   release.
 
-* Make sure to have required tools installed: namely Debugging tools for
-  Windows, a C compiler (PellesC or VC6) and Microsoft Platform SDK.
+* Make sure to have required tools installed:
+
+  - Debugging tools for Windows
+  - Microsoft Visual C++ 6.0 -- Note that newer versions do not work, and
```

Comment:
"and nor" sounds strange to me; I'd use either "and neither" or just "nor".

Reply:
Done.